app/templates/items/detail/tabs/_item_options.hbs

```diff
@@ -0,0 +1,130 @@
+  <button class="button options-arrow">
+    <div class="icon-holder">{{fa-icon 'caret-down' }}</div>
+  </button>
+
+  <div class="options_menu_list {{if displayItemOptions '' 'hidden'}}">
+    {{#each itemActions as |action|}}
+    <div class="option_menu">
+      <a {{action "triggerItemAction" item action.name bubbles=false}}>
+        {{fa-icon action.icon }}
+        {{action.name}}
+      </a>
+    </div>
+    {{/each}}
+  </div>
+</a>
+
+{{#popup-overlay open=readyForAction}}
+  <div class="dispatch-form package-actions-form">
+
+    <!-- TABS -->
+    <div class="row tab-container gc-orders-package-block">
+
+      <!-- DETAILS -->
+      <div class="columns small-3 sub-block main">
+        <div class="text-block">
+          <div class="line icon-holder">{{fa-icon (if item.isSet 'tags' (storage-icon item.storageTypeName)) size="lg"}}</div>
+          <div class="line text-holder">{{ item.inventoryNumber }}</div>
+        </div>
+      </div>
+
+      <!-- LOCATION -->
+      <div class="columns small-3 sub-block main">
+        <div class="text-block">
+          <div class="line icon-holder">{{fa-icon 'map-marker-alt' size="lg"}}</div>
+          <div class="line text-holder">
+            {{#if item.hasMultiLocations}}
+            {{t "item_status_bar.multiple"}}
+            {{else}}
+            {{#if item.firstAllLocationName}}
+            {{item.firstAllLocationName}}
+            {{else}}
+            N/A
+            {{/if}}
+            {{/if}}
+          </div>
+        </div>
+      </div>
+
+      <div class="columns small-6 tab sub-block main">
+        <div class="text-block row divided">
+          <div class="subtab column small-4">
+            <div class="line icon-holder">{{fa-icon 'paper-plane' size="lg"}}</div>
+            <div class="line text-holder"> {{ item.totalDispatchedQty }} </div>
+          </div>
+          <div class="subtab column small-4">
+            <div class="line icon-holder">{{fa-icon 'shopping-basket' size="lg"}}</div>
+            <div class="line text-holder"> {{ item.totalDesignatedQty }} </div>
+          </div>
+          <div class="subtab column small-4">
+            <div class="line icon-holder">{{fa-icon (if item.allowWebPublish 'eye' 'eye-slash') size="lg"}}</div>
+            <div class="line text-holder"> {{ item.availableQty }} </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="order-preview">
+      <div class="row gc-orders-package-block">
+        <div class="columns small-3 large-2 left">
+          <img src={{item.thumbImageUrl}} class="thumb" />
+        </div>
+
+        <div class="columns small-9 large-10 sub-block main" >
+          <div class="text-block">
+            <div class="line">{{ item.code.name }}</div>
+            <div class="line">{{ item.notes }}</div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="form-container">
+      <div class="title">{{fa-icon actionIcon}} {{actionName}}</div>
```

Comment:
I think we are missing not using `form` tag for making forms our code. Even I've being doing this mistake.  Without using `form` tag we miss few form experiences like submitting of form with just hitting enter. We are even missing `label` tags instead we are adjusting it with `div` or `span`. These are minor things but we are missing lots of user experiences.